qa-tests-backend/src/test/groovy/CertRotationTest.groovy

```diff
@@ -176,6 +176,7 @@ class CertRotationTest extends BaseSpecification {
         return true
     }
 
+    // This test is skipped in Operator and Helm, upgrader only works in Manifest based installations
     def "Test sensor cert rotation with upgrader succeeds for non-Helm clusters"() {
```

Comment:
Would it make sense to rephrase this to 
```suggestion
    def "Test sensor cert rotation with upgrader succeeds for manifest based installed clusters"() {
```

(if yes, also change line 181)

Reply:
done